tests/src/test/scala/org/http4s/EntityEncoderSpec.scala

```diff
@@ -1,64 +1,67 @@
 package org.http4s
 
-import scala.concurrent.Future
-
-import java.io.{StringReader, ByteArrayInputStream, FileWriter, File}
+import java.io._
 import java.nio.charset.StandardCharsets
 
 import cats._
+import cats.effect.IO
 import fs2._
 import org.http4s.headers._
 
+import scala.concurrent.Future
+
 class EntityEncoderSpec extends Http4sSpec {
   "EntityEncoder" should {
     "render strings" in {
       writeToString("pong") must_== "pong"
     }
 
     "calculate the content length of strings" in {
-      implicitly[EntityEncoder[String]].toEntity("pong").map(_.length) must returnValue(Some(4))
+      EntityEncoder[IO, String].toEntity("pong").map(_.length) must returnValue(Some(4))
     }
 
     "render byte arrays" in {
       val hello = "hello"
       writeToString(hello.getBytes(StandardCharsets.UTF_8)) must_== hello
     }
 
+    /* TODO: fs2-0.10 futureEncoder is tricky
```

Comment:
The current implementation of futureEncoder uses Task.fromFuture. There is an equivalent in cats-effect in IO.fromFuture, but we don't have access to that when the effect type is unknown.

I still think it makes sense to have an included EntityEncoder[IO, Future[A]] somewhere, since that seems like a useful instance, but I'm not sure where to put it.